tests/mgbench/workloads/importers/importer_ldbc_bi.py

```diff
@@ -1,3 +1,15 @@
+# Copyright 2023 Memgraph Ltd.
+#
+# Use of this software is governed by the Business Source License
+# included in the file licenses/BSL.txt; by using this file, you agree to be bound by the terms of the Business Source
+# License, and you may not use this file except in compliance with the Business Source License.
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0, included in the file
+# licenses/APL.txt.
+
+# ---  This is not an implementation of an LDBC Benchmark ---
```

Comment:
Put like the following `DISCLAIMER: This is NOT an official implementation of an LDBC Benchmark.`